### ISSUE TITLE
Mess up with network config

### ISSUE THREAD

**Issue by dchirikov** [clustervision/trinity#657]
## Description

On controller
br100 has 172.16.255.254/32 assigned as floating IP for login-a
```
# route -n | grep 172
172.16.0.0      0.0.0.0         255.255.240.0   U     0      0        0 br100
```
which give range 172.16.0.1 - 172.16.15.254. Note that floating IP is not included in this range.
Docker nodes (cXXX) have IPs in network 172.16.0.0/16:
```
[root@c217 /]# ip a
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN 
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
8: eth1@if2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UNKNOWN 
    link/ether 96:34:4c:4b:a2:78 brd ff:ff:ff:ff:ff:ff link-netnsid 0
    inet 172.16.0.217/16 scope global eth1
       valid_lft forever preferred_lft forever
    inet6 fe80::9434:4cff:fe4b:a278/64 scope link 
       valid_lft forever preferred_lft forever
```
If one manually assigns ip  172.16.16.1/16 to any host login-a unresponsive, despite prefix 16 makes this configuration valid.
```
[root@login01 ~]# ip a add 172.16.16.1/16 dev eno1
[root@login01 ~]# ping -c 1 172.16.0.217
PING 172.16.0.217 (172.16.0.217) 56(84) bytes of data.
64 bytes from 172.16.0.217: icmp_seq=1 ttl=64 time=0.254 ms

--- 172.16.0.217 ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 0.254/0.254/0.254/0.000 ms
[root@login01 ~]# ping -c 1 172.16.255.254
PING 172.16.255.254 (172.16.255.254) 56(84) bytes of data.

--- 172.16.255.254 ping statistics ---
1 packets transmitted, 0 received, 100% packet loss, time 0ms

```
From other hand, assigning 172.16.15.251 makes login-a pingable:
```
[root@login01 ~]# ip a del 172.16.16.1/16 dev eno1
[root@login01 ~]# ip a add 172.16.15.251/16 dev eno1
[root@login01 ~]# ping -c 1 172.16.255.254
PING 172.16.255.254 (172.16.255.254) 56(84) bytes of data.
64 bytes from 172.16.255.254: icmp_seq=1 ttl=63 time=0.416 ms

--- 172.16.255.254 ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 0.416/0.416/0.416/0.000 ms
[root@login01 ~]# ping -c 1 172.16.0.217
PING 172.16.0.217 (172.16.0.217) 56(84) bytes of data.
64 bytes from 172.16.0.217: icmp_seq=1 ttl=64 time=0.192 ms

--- 172.16.0.217 ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 0.192/0.192/0.192/0.000 ms
```
In other words, it is expected that all IPs from 172.16.0.0/16 are valid, because docker images use this range, but it turns out that you can only use 172.16.0.0/20.
Login-a needs to use gateway (even on the same node) to reach compute nodes in vc-a. which, apparently, rely on network performance.

## How to reproduce.

Install Trinity.

## Expected result.

Make network config consistent. Correct prefixes and networks. Assign IP for login-a in the same range as nodes for the virtual cluster "A".

**Comment by hansthen** [comment]
Unable to reproduce:

**Version:** r8 3bb0b2d
**Environment:** dmaster
```
[root@controller-1(dmaster*) ~]# route -n | grep 172
172.16.0.0      0.0.0.0         255.240.0.0     U     0      0        0 eth1
```

**Comment by dchirikov** [comment]
Ok. Cool! Happy to know. Any suggestion which commit fixed that? And how to fix on existing clusters?
```
# ip r | grep 172
172.16.0.0/20 dev br100  scope link
```-
And it's bit suspicious that you have `eth1`, not `br100`. Seems we have not the same configs.

**Comment by hansthen** [comment]
The commit that fixed it was b57f3a3a, which is included in the R7 we released, but not in the version you have. There is a unit file trinity-route in R7 that sets the route on the bridge. You need to change the netmask in that file.

In R8 we split the `br100` and used a separate interface (`eth1`) for vm/container traffic in order to make network traffic more clean.
